admin_solutions/user_role_mgmt.adoc

```diff
@@ -0,0 +1,542 @@
+than the one you currently are in.
+
+[[limit-number-projects]]
+=== Limiting the Number of Projects a User Can Have
+
+You can
+link:../admin_guide/managing_projects.html#limit-projects-per-user[limit the number of projects] that a user may request by categorizing users with label selectors with the `oc label` command. A label selector consists of the label name and the label value:
+
+----
+label=value
+----
+
+Once users are labeled, you must
+link:../admin_guide/managing_projects.html#modifying-the-template-for-new-projects[modify the default project template] in the *_master-config.yaml_* file
+link:../architecture/additional_concepts/admission_controllers.html[using an admission control plug-in]. This allows some users to create more projects than others, and you can define different values (or levels) for each label.
+
+*Use Case: You want to define 3 different levels for your users which define how many projects they can request.* The label is named `level`, and the possible values are  `bronze`, `silver`, `gold`, and `platinum`. Platinum users do not have a maximum number of project requests, gold users can request up to 10 projects, silver users up to 7 projects, bronze users up to 5 projects, and any users without a label are by default only allowed 2 projects.
+
+. Apply label selectors to users. For example, to apply the `level` label selector with a value of `bronze`:
++
+----
+$ oc label user <user_name> level=bronze
+----
++
+Repeat this step for the other levels.
++
+[NOTE]
+====
+Each user can only have one value per label. For example, a user cannot be both `gold` and `silver` for the level label. However, when configuring the *_master-config.yaml_* file, you could select users that have any value for a label with a wildcard; for example, `level=*`.
+====
++
+. Optionally, verify the previous step by viewing the list of labeled users for each value:
++
+----
+$ oc get users -l level=bronze
+$ oc get users -l level=silver
+$ oc get users -l level=gold
+$ oc get users -l level=platinum
+----
++
+If you need to remove a label from a user to make a correction:
++
+----
+$ oc label user <user_name> level-
+----
+. Modify the *_master-config.yaml_* file to define project limits for this label with the numbers stated in this use case. Find the `admissionConfig` line and create the configuration below it:
++
+[source,yaml]
+----
+admissionConfig:
+  pluginConfig:
+    ProjectRequestLimit:
+      configuration:
+        apiVersion: v1
+        kind: ProjectRequestLimitConfig
+        limits:
+        - selector:
+            level: platinum
+        - selector:
+            level: gold
+          maxProjects: 10
+        - selector:
+            level: silver
+          maxProjects: 7
+        - selector:
+            level: bronze
+          maxProjects: 5
+        - maxProjects: 2
+----
++
+. After modifying the *_master-config.yaml* file, you must restart the master for changes to take effect.
```

Comment:
Probably not necessary to mention the first part:

"Restart the master host for the changes to take effect:"